english/Raiders/LetsgoRaiders.md

```diff
@@ -1,3 +1,6 @@
 Lets go Raiders!
 
-[Here's the link](https://www.google.com)
\ No newline at end of file
+[Here's the link](https://www.google.com)
+
+You see Batman at the Raiders game you are attending and you decide
+to go ask him some personal questions. [This is what you find](https://github.com/udacity/create-your-own-adventure/blob/master/english/batman/batman.md)!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.